src/directory/directory.mjs

```diff
@@ -1644,57 +1653,50 @@ export const directory = {
               path: 'src/pages/[platform]/sdk/auth/index.mdx',
               children: [
                 {
-                  path: 'src/pages/[platform]/sdk/auth/custom-auth-flow/index.mdx'
+                  path: 'src/pages/[platform]/sdk/auth/getting-started/index.mdx'
                 },
                 {
-                  path: 'src/pages/[platform]/sdk/auth/device-features/index.mdx'
+                  path: 'src/pages/[platform]/sdk/auth/guest-access/index.mdx'
                 },
                 {
                   path: 'src/pages/[platform]/sdk/auth/drop-in-auth/index.mdx'
                 },
                 {
-                  path: 'src/pages/[platform]/sdk/auth/federated-identities/index.mdx'
-                },
-                {
-                  path: 'src/pages/[platform]/sdk/auth/getting-started/index.mdx'
+                  path: 'src/pages/[platform]/sdk/auth/working-with-api/index.mdx'
                 },
                 {
-                  path: 'src/pages/[platform]/sdk/auth/guest-access/index.mdx'
+                  path: 'src/pages/[platform]/sdk/auth/federated-identities/index.mdx'
                 },
                 { path: 'src/pages/[platform]/sdk/auth/hosted-ui/index.mdx' },
                 {
-                  path: 'src/pages/[platform]/sdk/auth/working-with-api/index.mdx'
-                }
-              ]
-            },
-            {
-              path: 'src/pages/[platform]/sdk/configuration/index.mdx',
-              children: [
+                  path: 'src/pages/[platform]/sdk/auth/custom-auth-flow/index.mdx'
+                },
                 {
-                  path: 'src/pages/[platform]/sdk/configuration/setup-options/index.mdx'
+                  path: 'src/pages/[platform]/sdk/auth/device-features/index.mdx'
                 }
               ]
             },
-            {
-              path: 'src/pages/[platform]/sdk/info/index.mdx',
-              children: [
-                {
-                  path: 'src/pages/[platform]/sdk/info/app-uninstall/index.mdx'
-                },
-                { path: 'src/pages/[platform]/sdk/info/overview/index.mdx' }
-              ]
-            },
+            // Uncommented because this feature is not shown
```

Comment:
@salihgueler Looks like these pages only exsit for Swift - https://docs.amplify.aws/sdk/info/overview/q/platform/ios/ Can you check that they were imported and have the correct platform filter on them? Once that is done I think this can be unhidden

Reply:
Can we then limit this to swift here as well? Reverted the comment btw.